test/appStartup.test.ts

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as coreTypes from '@azure/functions-core';
```

Comment:
Overall tests look good! 🎉 It's a bit confusing which tests go in appStartup.test.ts vs workerInitHandler.test.ts vs functionEnvironmentReloadHandler.test.ts, but I don't have a better suggestion. Perhaps something we improve in the future